### ISSUE TITLE
checkpoint release

### ISSUE THREAD

**Issue by jusepv** [mao1207/BioMed-VITAL#1]
Thanks for the great research.

Do you have any plans to release a checkpoint of the model you trained?

**Comment by mao1207** [comment]
Yes, we have already released our model checkpoint. You can access it through this [link](https://huggingface.co/mao1207/BioMed-VITAL-models).
